reco_utils/recommender/rbm/rbm.py

```diff
@@ -0,0 +1,973 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The rbm tests are missing. This is how we did the tests in SAR: https://github.com/Microsoft/Recommenders/blob/staging/tests/unit/test_sar_singlenode.py.
Let me know if you need help on how to structure the tests

Reply:
Thanks, my question is as follows: The class contains several methods, do they need individual tests? If so, one probably need to use TF eager execution or create several sessions inside the test. I cannot find the unit test for the ncf algo that also uses TF, there is only `test_ncf_dataset`. It would be good to see how the test has been written in that case.